### DISCUSSION TITLE
doris docker connect to FE error：ERROR 2002 (HY000): Can't connect to MySQL server on '127.0.0.1' (115)

### DISCUSSION THREAD

**Discussion by chenyb868** [apache/doris#36433] [A - General / Q&A]
1、当前的doris集群采用docker部署，部署的方式参照官方文档(https://doris.apache.org/zh-CN/docs/install/cluster-deployment/run-docker-cluster)，部署了3个FE、3个BE；

2、部署完成后，第一次采用 **mysql -uroot -P9030 -h127.0.0.1** 连接FE，可正常连接；

3、**docker容器重启后**，再次采用**mysql -uroot -P9030 -h127.0.0.1** 连接FE，无法连接，报错信息为：**ERROR 2002 (HY000): Can't connect to MySQL server on '127.0.0.1' (115)**

针对上述问题，该如何解决……

**Comment by ixzc** [comment]
中文问题，建议去ask.selectdb.com去问问题

**Reply by chenyb868** [reply]
已上ask.selectdb.com去问，帮忙看下这问题怎么解
